docs/src/explanations/explanations.md

```diff
@@ -27,8 +27,9 @@ read these documents in the following order:
 * [Sequential Circuits](sequential-circuits)
 * [Memories](memories)
 * [Interfaces and Connections](interfaces-and-connections)
+* [Deep Dive into Connection Operators](connection-operators)
```

Comment:
Upon reading over this, the fact that this article talks about `DontCare` makes me think we should move it to the end of this ToC because DontCare is not introduced until "UnconnectedWires". I will push a change to do that